python/taichi/type/annotations.py

```diff
@@ -8,12 +8,24 @@ class ArgAnyArray:
         element_dim (Union[Int, NoneType], optional): None if not specified (will be treated as 0 for external arrays), 0 if scalar elements, 1 if vector elements, and 2 if matrix elements.
```

Comment:
docstring?

Reply:
Nice catch! I'll send a separate PR for this to avoid rerunning CI for all 6 PRs :P